index.html

```diff
@@ -6,9 +6,12 @@
         <!-- CDN reset -->
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
 
-        <!-- linked stylesheet below -->
-        <link rel="stylesheet" href="css/style.css">
-    </head>
+    <!-- linked stylesheet below -->
+    <link rel="stylesheet" href="css/style.css">
+
+    <title>SavvyCoders Books, LLC.</title>
```

Comment:
There's already a `<title>` tag on line 5. Please modify that one instead of adding these!